jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -1315,6 +1317,25 @@ def _make_scatter_harness(name,
                         scatter_indices=np.array([[4], [77]]),
                         update_shape=(2,))
 
+# Validate no XLA scatters
+for dtype in set(jtu.dtypes.all) - set(jtu.dtypes.complex) - set(jtu.dtypes.boolean):
+  for f_lax in [
```

Comment:
I would add `lax.scatter` here but I get
```python
E       AssertionError: Harness group 'scatter' must be either part of 'harness_groups_no_limitations' or must have a custom Jax2TfLimitation.classmethod defined (see module docstring)
```
when I try to. 

Next commit fixes this.